fakecam/fake.py

```diff
@@ -221,7 +225,7 @@ def hologram_effect(self, img):
         # add a blue tint
         holo = cv2.applyColorMap(img, cv2.COLORMAP_WINTER)
         # add a halftone effect
-        bandLength, bandGap = 2, 3
+        bandLength, bandGap = 3, 4
```

Comment:
does this change really fix or improve anything? It seems a little out of place, especially as you are doing the background blur in this commit :)

I do think it may be a good idea to have this as commandline options though, if it is desireable to be tweaked (as would be the color(map)